### PR TITLE
#846 - updated docs for IFTTT integration with openHAB

### PR DESCRIPTION
**Pull request by Maurits28**
Addresses issue [#846](https://github.com/openhab/openhab-docs/issues/846
)

Some remarks:
- Removed information on DO button, as this was integrated into the core IFTTT app
- Changed channels to services and recipes to applets (new names from IFTTT)
- Updated links
- Removed outdated images containing old openHAB logo

Would like to add some images (one logo at the start) and some examples at the end, but not a clue how to do this.

Would appreciate input on this initial change

### CODE REVIEW

**Comment by ghys** [comment]
Thank you.
Note that the IFTTT article was previously merely a "marketing" news article, part of the old website, not actual docs. That's in part why it was left here - we can merge here as a short-term solution (once you finish your changes) but ultimately this page should be moved to openhab-docs.

**Comment by Maurits28** [comment]
Perfect, I don't know how to move it, and indeed it was a bit of a marketing story. Awaiting some comments on my earlier remarks, once finalized please move it to the appropriate location.

**Comment by Confectrician** [comment]
Lets improve it and i take core of the move with @ysc afterwards. :)

**Comment by Maurits28** [comment]
I made a few minor changes, but there were no more comments on the post in the [forum](https://community.openhab.org/t/draft-update-ifttt-docs/63301).

I think it can be merged, @Confectrician can you add an image to the page of the openHAB/IFTTT logo?

screen shot 2019-01-10 at 19 30 25

File: docs/ecosystem/ifttt/readme.md [hunk 1]
```diff
@@ -11,71 +11,48 @@ meta:
 
 # IFTTT Integration for openHAB
 
-<div class="img-wrapper"><img src='./images/ifttt-meets-openhab.jpg' alt='IFTTT meets openHAB'/></div>
+[If This Then That (IFTTT)](https://ifttt.com) is a very popular web-based service that allows users to create "applets", which can combine different web services into powerful automation rules. With its [hundreds of services](https://ifttt.com/services), there are limitless new creative options: Be notified through text-to-speech if you are receive a DM on twitter, be warned through your lights if your favorite stock price falls below a certain threshold or put your house in away-mode when you turn on the ignition of your car - you can get further inspiration by browsing through existing applets.
```
Comment (hakan42) [inline docs/ecosystem/ifttt/readme.md]: Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.

File: docs/ecosystem/ifttt/readme.md [hunk 2]
```diff
@@ -11,71 +11,48 @@ meta:
...
-      <section class="img-wrapper">
-        <img alt="" src="./images/ifttt1.png" style="-webkit-transform: scale(1);">
-      </section>
-    </a>
+**1. Connect openHAB with IFTTT**
 
-  <div class="da-animate da-slideFromTop" style="display: block;">
-    <span class="iconWrapper iconLink icon-search" style="font-style: italic; margin-top: 63px"></span>
-  </div></a>
+IFTTT is available to all our users through the myopenHAB service. Activating the integration is easy. Just log in to your IFTTT account and activate the [openHAB service](https://ifttt.com/openhab). You will be forwarded to the [myopenHAB.org website](http://www.myopenhab.org/) to authorize the IFTTT service connection.
```
Comment (hakan42) [inline docs/ecosystem/ifttt/readme.md]: Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.

File: docs/ecosystem/ifttt/readme.md [hunk 3]
```diff
@@ -11,71 +11,48 @@ meta:
...
-  </article>
+**2. Expose specific items to IFTTT**
 
-  <article class="span4 bloc">
-    <a href="https://ifttt.com/recipes/297847-open-garage-when-i-turn-on-ignition">
-      <section class="img-wrapper">
-        <img alt="" src="./images/ifttt2.png" style="-webkit-transform: scale(1);">
-      </section>
-    </a>
+Before you start creating IFTTT applets you need to make sure that you configured your local openHAB runtime to expose certain items to myopenHAB.org. Only those items will be visible to IFTTT. Read here [how to expose items to myopenHAB.org](https://www.openhab.org/addons/integrations/openhabcloud/#configuration). You will also be able to send commands to those items from IFTTT Applets.
```
Comment (hakan42) [inline docs/ecosystem/ifttt/readme.md]: Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.

File: docs/ecosystem/ifttt/readme.md [hunk 4]
```diff
@@ -11,71 +11,48 @@ meta:
...
-Besides the new kinds of integration possibilities, IFTTT is also a perfect start for new openHAB users as it makes the creation of rules dead simple - no need to do any scripting, no complex setup routines, it all works through a few mouse clicks!
-
-<div class="img-wrapper"><img src='./images/ifttt-do.jpg' alt='IFTTT Do Button'/></div>
-
-We have also integrated with IFTTT’s [Do Button](https://ifttt.com/products/do/button). The Do button allows you to access often required functionality very quickly - either through the Do app, as a widget or even as a floating button, that is always present on your screen.
-
-## Let's Go!
-
-IFTTT is available to all our users through the [myopenHAB service](http://www.myopenhab.org/). Once you have your myopenHAB account set up, all you have to do is to decide, which items you want to share with IFTTT and to activate the openHAB channel.
+If you are using iOS or Android, you can use IFTTT to send notifications from your openHAB runtime to your device running iOS or Android. Read more about this [here](https://community.openhab.org/t/openhab-send-sensor-notification-to-ios-android-using-ifttt/24725).
```
Comment (hakan42) [inline docs/ecosystem/ifttt/readme.md]: Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.

File: docs/ecosystem/ifttt/readme.md (hunk 1 above)
Comment (Maurits28) [reply docs/ecosystem/ifttt/readme.md]: Hi Hakan, thanks for this, I changed it. No problem. Can you explain what difference between the two is?

Comment (Maurits28) [reply docs/ecosystem/ifttt/readme.md]: Hi Hakan, no problem, I changed it. Would you mind explaining what the difference is?

Comment (hakan42) [reply docs/ecosystem/ifttt/readme.md]: If you change something small in one sentence, only that sentence is being shown as a diff in github / "git diff" / whatever too you use.

Basically, makes reviewing easier 😄

**Comment by Maurits28** [comment]
> :shipit:

??

**Comment by Confectrician** [comment]
This is called `:shipit:`.
Just an emoji for "let go that merge". 🙂

**Comment by Confectrician** [comment]
@ghys  would you do me a favor and merge this one?
I will take care or the move to openhab-docs afterwards.

**Comment by ghys** [comment]
Sorry I'm busy with family stuff these days, this slipped off my mind.

**Comment by Confectrician** [comment]
Thanks. No need to hurry or apologize. 🙂 
We are all here in our free time, so sometimes things have to wait, that's perfectly fine.

**Pull request merged**